resources/public/api/conf/common/testing.md

```diff
@@ -1,5 +1,5 @@
 You can use the sandbox environment to [test this API](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing).
 
-It does not currently support [stateful behaviour](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing/stateful-behaviour), but you can use the payloads described in Resources to test specific scenarios.
+It does not currently support [stateful behaviour](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing/stateful-behaviour), but you can use the payloads described in Test data sections of each endpoint to test specific scenarios.
```

Comment:
Instead of using ** as was done in the yaml file, surround test data with strong tags i.e `<strong>Test data</strong>` to highlight the text semantically please